src/Orleans.CodeGeneration.Build/Orleans.CodeGeneration.Build.csproj

```diff
@@ -9,6 +9,7 @@
   <PropertyGroup>
     <TargetFramework />
     <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
```

Comment:
will this `OS` property be set on local builds or is this a CI server-specific value?

Reply:
I think that's coming from MSBuild itself, default SDK targets/props also using this property to check for OS.